tests/testAllParallel.nim

```diff
@@ -2,21 +2,25 @@
 #   ./dht/[test_providers, test_providermngr],
 #   ./discv5/[test_discoveryv5, test_discoveryv5_encoding]
 
-import osproc
+import std/[os, osproc, strutils]
+
+## we wanna get nimble paths if set
+let nimblePaths = getEnv("NIMBLE_PATHS", "").split($PathSep & $PathSep).join(" ")
```

Comment:
@yyoncho I ended up doing this to get the Nimble + getPathsClause() to work when running parallel tests. `testAllParallel.nim` compiles and runs each test since NimScript doesn't support parallel exec (that I know of). It'd be nicer to do the below in a `build.nims` or nimble file.

Reply:
If there's a better method let me know!